.github/workflows/ci.yml

```
@@ -123,4 +123,4 @@ jobs:
     - name: "Upload coverage to Codecov"
       uses: codecov/codecov-action@v2
       with:
-        fail_ci_if_error: true
+        fail_ci_if_error: ${{ github.ref_name == 'develop' && 'true' || github.ref_name == 'main' && 'true' || 'false' }}
```

Do we need the `&&`s anymore?
```suggestion
        fail_ci_if_error: ${{ github.ref_name == 'develop' || github.ref_name == 'main' }}
```

---

lets see!